BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/GiniBankNetworkingScreenApiCoordinator.swift

```diff
@@ -492,12 +497,40 @@ extension GiniBankNetworkingScreenApiCoordinator {
                 self.resultsDelegate?.giniCaptureAnalysisDidFinishWith(result: result)
 
                 self.giniBankConfiguration.skontoDiscounts = result.skontoDiscounts
+                self.sendSkontoFeedback(extractions: extractions)
             } else {
                 analysisDelegate?.tryDisplayNoResultsScreen()
                 self.documentService.resetToInitialState()
             }
         }
     }
+    
+    private func sendSkontoFeedback(extractions: [String: Extraction]) {
+        guard let amountToPay = extractAmountToPay(from: extractions) else { return }
+        let amountToPayString = formatAmountToPay(amountToPay)
+        let amountExtraction = createAmountExtraction(value: amountToPayString)
+        self.giniBankConfiguration.sendTransferSummaryWithSkonto(amountExtraction: amountExtraction, amountToPayString: amountToPayString)
+    }
+    
+    private func formatAmountToPay(_ amount: ExtractionAmount) -> String {
+        let formattedValue = amount.value.stringValue(withDecimalPoint: 2) ?? "\(amount.value)"
+        return "\(formattedValue):\(amount.currency.rawValue)"
+    }
+
+    private func extractAmountToPay(from extractions: [String: Extraction]) -> ExtractionAmount? {
+        guard let amountValue = extractions["amountToPay"]?.value,
+              let amountComponents = amountValue.split(separator: ":").first,
+              let value = Decimal(string: String(amountComponents)) else { return nil }
+        return ExtractionAmount(value: value, currency: .EUR)
+    }
+    
+    private func createAmountExtraction(value: String) -> Extraction {
+        return Extraction(box: nil,
+                          candidates: nil,
+                          entity: "amount",
+                          value: value,
+                          name: "amountToPay")
+    }
```

Comment:
[suggestion] Can we somehow separate this logic outside of this file? it seems something specific to Skonto and maybe we can have an extension dedicated to this. What do you say? And if we can include other things as well, that would be great since this file in the feature will be longer than it is now.